libclangext/src/main/cpp/ExtVector.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include <cassert>
+
+#include <clang/AST/Type.h>
+#include <clang-c/Index.h>
+
+
+using namespace clang;
+
+
+static inline QualType GetQualType(CXType CT) {
```

Comment:
Why not reuse `unwrapCXType` from ClangExt.cpp instead?

Reply:
Calling getFromOpaquePtr direrctly would be even simpler. However, the intention is to make copy-paste in clang_isExtVectorType to be easily comparable with prototype in Type.cpp. So I keep a snippet intact,  including naming.
Second: unwrapCXType declared as static, which implies two options: either refactor cpp, h and build script to reuse declaration; or add content to ClangExt.cpp. Refactoring efforts are hardly paid back in this case and both options contradict the intention to remove a kludge as soon as libclang is upgraded to 59 version (clang 9).